vespalib/src/vespa/vespalib/datastore/fixed_size_hash_map.h

```diff
@@ -99,9 +119,26 @@ class FixedSizeHashMap {
     FixedSizeHashMap(uint32_t module, uint32_t capacity, uint32_t num_shards, const FixedSizeHashMap &orig, const EntryComparator& comp);
     ~FixedSizeHashMap();
 
-    KvType& add(const EntryComparator& comp, EntryRef key_ref, std::function<EntryRef(void)>& insert_entry);
-    KvType* remove(const EntryComparator& comp, EntryRef key_ref);
-    KvType* find(const EntryComparator& comp, EntryRef key_ref);
+    ShardedHashComparator getComp(const EntryComparator& comp) {
```

Comment:
@baldersheim consider using snake case (_get_comp_) to be in sync with the rest of the code.

Reply:
Fixed in #17273